website/content/docs/release-notes/v0_16_0.mdx

```diff
@@ -201,5 +201,55 @@ description: |-
     <a href="/boundary/tutorials/self-managed-deployment/upgrade-version">Upgrade to the latest version of Boundary</a>
     </td>
   </tr>
+  </tbody>
+</table>
+
+## Feature deprecations and EOL
+
+<table>
+  <thead>
+    <tr>
+      <th style={{verticalAlign: 'middle'}}>EOL</th>
+      <th style={{verticalAlign: 'middle'}}>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+
+  <tr>
+    <td style={{verticalAlign: 'middle'}}>
+      <code>boundary daemon</code> command
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+     The <code>boundary daemon</code> command has been deprecated in favor of the new <code>boundary cache</code> command. The functionality remains the same.
+      <br /><br />
+      Learn more:&nbsp;
+      <a href="/boundary/docs/commands/cache"><code>boundary cache</code></a>
+    </td>
+  </tr>
+
+  <tr>
+    <td style={{verticalAlign: 'middle'}}>
+      <code>include_terminated</code> field removed
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+     The <code>include_terminated</code> field from the <code>list sessions</code> command has been deprecated and will be removed in an upcoming release. Boundary will return terminated sessions in all list session responses, unless they are filtered out.
```

Comment:
Just curious if it's worth mentioning how to filter it out

Reply:
Yep, I think that would be good info for users. I updated it and referenced the filter field and also included a link to our docs for filtering resources. Thanks for the review!